content/spin/kv-store-tutorial.md

```diff
@@ -22,27 +20,13 @@ url = "https://github.com/fermyon/developer/blob/main//content/spin/kv-store.md"
 
-## SQLite
-
-SQLite is an integral part of Spin's key/value API. As of Spin v0.9.0 onwards, users can easily access a built-in, _local_ key/value SQLite database in every Spin application. This persistent storage feature is available by default and with minimal configuration. [Supported language SDKs](https://developer.fermyon.com/spin/language-support-overview) currently include Go, JavaScript/TypeScript, Python and Rust.
-
-> Spin users can now persist and retrieve non-relational data from a key/value store across multiple requests to and from the same application; written in any of these languages.
-
-### Redis, PostgreSQL & SQLite
-
-A while back, around the Spin v0.5.0 version, we published an article on [Persistent Storage in Webassembly Applications](https://www.fermyon.com/blog/persistent-storage-in-webassembly-applications). In that previous article, we showed the Spin framework's capabilities of providing WebAssembly executables with access to different levels of on-disk persistence. We demonstrated manually installing Redis from source, running our Redis server on localhost and then reading and writing from Redis via both the Redis CLI itself and also via the Spin SDK. Our current documentation also has examples of using both [Redis](https://developer.fermyon.com/cloud/data-redis) and [PostgreSQL](https://developer.fermyon.com/cloud/data-postgres), via [Redis Labs](https://redis.com/) and [ElephantSQL](https://www.elephantsql.com/plans.html) services respectively. 
-
-In all of these previous examples, to persist data within your application, a separate data storage layer is a requirement. These methods of data persistence are perfectly fine and sound to use as part of your application, if you choose to do so.
-
-However, from Spin v0.9.0 onwards, you are no longer required to install any on-disk persistence outside of just installing Spin itself. SQLite is embedded inside Spin in a way that is analogous to how one would imagine using a software library. When you run your Spin application using the [spin up](https://developer.fermyon.com/common/cli-reference#up) command, SQLite is available to your application. When your application is no longer running, neither is SQLite. Your data will continue to persist at all times (including during restarts) and will support the running of your application.
-
-In the future, it may be possible to embed other database technologies into Spin. For example, future Spin SDK updates may target other databases such as Redis, which would allow you to build applications using Redis via minimal Spin configuration without the need to locally install and maintain your own Redis instance.
-
-Let's get started with creating and deploying your first Spin application that uses this new key/value storage mechanism.
+* Use the key value store SDK to get, set, and retrieve key value pairs
+* Configure your application manifest (`spin.toml`) to use the default key value store
```

Comment:
Move this to first in the list so that the roadmap here lines up with the order of things in the tutorial